script.sh

```diff
@@ -6,38 +6,59 @@
 #   dir2
 #   dir3
 #       dir4
+mkdir -p ./task/dir1
+mkdir ./task/dir2
+mkdir ./task/dir3
+mkdir ./task/dir3/dir4
 
 # изменяем текущую директорию на task
+cd task/
```

Comment:
Можно не указывать слеши
```suggestion
cd task
```